tests/Foundation/FoundationViteTest.php

```diff
@@ -3,34 +3,23 @@
 namespace Illuminate\Tests\Foundation;
 
 use Illuminate\Foundation\Vite;
-use Illuminate\Routing\UrlGenerator;
-use Illuminate\Support\Facades\Facade;
 use Illuminate\Support\Facades\Vite as ViteFacade;
 use Illuminate\Support\Str;
-use Mockery as m;
-use PHPUnit\Framework\TestCase;
+use Orchestra\Testbench\TestCase as TestbenchTestCase;
 
-class FoundationViteTest extends TestCase
+class FoundationViteTest extends TestbenchTestCase
```

Comment:
This brings the test case closer to reality so our tests reflect what is really going to happen.